packages/sanity/src/core/i18n/Translate.tsx

```diff
@@ -109,7 +108,13 @@ export function Translate(props: TranslationProps) {
   })
 
   const tokens = useMemo(() => simpleParser(translated), [translated])
-  const listFormat = useListFormat()
```

Comment:
Could we keep this and wrap it in a try/catch and use the en-US format if we catch an exception?

Reply:
good catch! pushed that logic to the useListFormat hook to maintain rule of hooks.